### ISSUE TITLE
weird result when evaluate with lora

### ISSUE THREAD

**Issue by ZhanHeshen** [OptimalScale/LMFlow#398]
I finetune the llama-7b model on alpaca by runing run_finetune_with_lora.sh for 0.01 epochs, and evaluate by run_evaluate_with_lora.sh. I only change the path without any modification to the setup, but the result is weird, the accuracy on the test set was 0.0, do anyone know why this would happen?

Here is the result of the last test sample:
![image](https://github.com/OptimalScale/LMFlow/assets/71877273/6006518b-91c2-4cef-b98b-07ea2a66e882)

**Comment by SHUMKASHUN** [comment]
Hi,
        alpaca is an  instruction-following dataset and the outputs are not supposed to be measure by accuracy. You can manually check if the prediction answers are satisfied. Or you can try finetune your model on other datasets like MedMCQA which can be measured by accuracy. Also, when fine-tuning your model, try increase your epoch number, otherwise the model can learn little about the training data.

**Comment by ZhanHeshen** [comment]
@SHUMKASHUN Thank you! now the result is reasonable.
